cluster/src/main/java/com/hartwig/pipeline/execution/vm/BashStartupScript.java

```diff
@@ -50,7 +50,7 @@ String asUnixString(StorageStrategy storageStrategy) {
                 "function die() {",
                 "  exit_code=$?",
                 "  echo \"Unknown failure: called command returned $exit_code\"",
-                format("  gsutil -m cp %s gs://%s", localLogFile, runtimeBucketName),
+                format("  gsutil -m cp %s gs://%s/results/", localLogFile, runtimeBucketName),
```

Comment:
I thought we had a requirement that it live in the top-level?

Reply:
don't think so, when the run succeeds this is where the logs go.